inst/help/auditBayesianEvaluation.md

```diff
@@ -140,7 +140,7 @@ The expected errors are the tolerable errors that can be found in the sample whi
 ### References
 ---
 - AICPA (2019). <i>Audit Guide: Audit Sampling</i>. American Institute of Certified Public Accountants.
-- Derks, K. (2023). jfa: Statistical Methods for Auditing. R package version 0.6.6.
+- Derks, K. (2023). jfa: Statistical Methods for Auditing. R package version 0.7.0.
```

Comment:
```suggestion
- Derks, K. (2023). jfa: Statistical Methods for Auditing. R package version 0.7.1.
```
since that is what the description lists as lower bound? But you do have a lot of instances of this reference... :sweat_smile:

Reply:
Actually, let's not get into the rabbit hole of changing the version numbers every time. I just made it "latest version" now :p